[Bug] Use Presets to introduce the printing function and report an error

### 在您提交此问题之前，您是否检查了以下内容？

- [x] 这真的是个问题吗？
- [x] 我已经在 Github Issues 中搜索过了，但没有找到类似的问题。

### 受影响的包和版本

0.8.3

### 复现链接

https://gitee.com/wang134246/univer_dome.git

### 预期行为

打印功能正常使用

### 实际行为

![Image](https://github.com/user-attachments/assets/316e7928-0020-4e41-88e7-c2b0528b0825)

### 运行环境

_No response_

### 系统信息

 Cannot find module '@univerjs/core'
如果vue.config.js文件中加了chainWebpack或configureWebpack就会出现该问题。不知是为什么会发生这种问题。

--- a/common/shared/vite/utils.ts
+++ b/common/shared/vite/utils.ts
@@ -54,7 +54,9 @@ export function obfuscator(): Plugin {
             for (const file in bundle) {
                 if (bundle[file].type === 'chunk' && /\.js$/.test(file)) {
                     const code = bundle[file].code;
-                    const obfuscationResult = JavaScriptObfuscator.obfuscate(code);
+                    const obfuscationResult = JavaScriptObfuscator.obfuscate(code, {
+                        ignoreImports: true,
+                    });
                     bundle[file].code = obfuscationResult.getObfuscatedCode();
                 }
             }

